iree/turbine/kernel/wave/utils.py

```diff
@@ -1052,8 +1052,18 @@ def all_equal(input_list: list[Any]) -> bool:
     return all(elem == input_list[0] for elem in input_list)
 
 
+DEFAULT_GPU_DEVICE = None
```

Comment:
Rather than have this as a global variable, can we instead pass this as an argument to the get_gpu_device_name(int) function? I am trying to see if we can reduce the number of command line arguments we need to export and this will also help when we get to distributed kernels (across multiple GPUs).

Reply:
After a discussion, it makes sense to keep this for now to minimize changes across tests. But we will remove the WAVE_DISTRIBUTE_GPU_TESTS parameter and instead make it a pytest argument.